tests/index.js

```diff
@@ -1,5 +1,7 @@
 var test = require('tape');
 
+const serverSide = require('./server/index.js');
+
 test('should return -1 when the value is not present in Array', function (t) {
```

Comment:
@esraajb did you mean the serverSide tests ? or the basic test in the main index.js?